crates/runtime/src/http/v1/mod.rs

```diff
@@ -93,30 +94,50 @@ impl utoipa::IntoParams for Format {
 #[derive(Default, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "lowercase")]
 /// The various formats that the Arrow data can be converted and returned from HTTP requests.
-pub enum ArrowFormat {
+pub enum ResponseMimeType {
     #[default]
     Json,
     Csv,
     Plain,
+    VndJsonV1,
```

Comment:
```suggestion
    VndSqlJsonV1,
```

Reply:
Will apply this suggestion as the next iteration (schema support) - Thank you!